mobile/build.gradle

```diff
@@ -1,18 +1,18 @@
 buildscript {
     repositories {
-        maven { url 'http://download.crashlytics.com/maven' }
         mavenCentral()
+        maven { url 'https://maven.fabric.io/public' }
     }
 
     dependencies {
-        classpath 'com.crashlytics.tools.gradle:crashlytics-gradle:1.14.3'
         classpath 'com.github.jcandksolutions.gradle:android-unit-test:2.1.1'
+        classpath 'io.fabric.tools:gradle:1.+'
```

Comment:
Hardcode version `1.14.4`. The reason being, a year form now this build would be non-reproducible if you use the +. So using the + (which gets the latest) is discouraged.